jobs/hardware_report.py

```diff
@@ -0,0 +1,1009 @@
+import json
```

Comment:
Can you refer to the original location of the script from the mozetl repository at the top of the file, where the tests are also located?

Reply:
I put a link on line 27.